src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -96,10 +96,14 @@
   [{default-on-confirm       :on-confirm
     default-limit-crypto     :limit-crypto
     default-crypto-decimals  :crypto-decimals
+    type                     :type
```

Comment:
if possible we use a different name than `type` here because it is actually a cljs function.
`transfer-type` is better? 🤔